packages/ds-react-core/src/ui/Chip/Chip.css

```diff
@@ -0,0 +1,79 @@
+/* @canonical/generator-canonical-ds 0.4.0-experimental.0 */
+
+.ds.chip {
+  border-color: var(--intent-color-border, var(--chip-color-border));
+  border-style: solid;
+  border-radius: var(--chip-border-radius);
+  border-width: var(--chip-border-width);
+
+  color: var(--intent-color-text-tinted, var(--chip-color-text));
+  background-color: var(--intent-color-tinted, var(--chip-color-background));
+
+  display: inline-flex;
+  align-items: baseline;
+
+  user-select: none;
+  white-space: nowrap;
+
+  font-size: var(--chip-font-size);
+  font-weight: var(--chip-font-weight);
+  line-height: var(--chip-line-height);
+
+  margin-block-end: var(--chip-margin-bottom);
+  margin-inline-end: var(--chip-margin-left);
+  padding-block: var(--chip-padding-vertical);
+  padding-inline: var(--chip-padding-horizontal);
+
+  & > .lead {
+    font-variant-caps: all-small-caps;
+    font-variant-numeric: oldstyle-nums;
+  }
+
+  & > .lead + .value::before {
+    content: ": ";
+  }
+
+  & > .lead,
+  & > .value {
+    display: inline;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+
+  & > .dismiss {
+    background-image: url("data:image/svg+xml,%3Csvg width='16' height='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill='%23000' fill-rule='nonzero' d='M13.041 1.898l1.06 1.06L9.062 8l5.04 5.042-1.06 1.06L8 9.062 2.96 14.1l-1.06-1.06L6.938 8 1.9 2.96l1.06-1.06 5.04 5.04z'/%3E%3C/svg%3E");
```

Comment:
Once we have icons components we would need to refactor this, but is fine for now.